config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -39,6 +39,7 @@ en:
             auto_grade: 'Auto Grade Submission'
             evaluate_answers: 'Evaluate Answers'
             unsubmit: 'Unsubmit Submission'
+            unmark: 'Ungrade Submission'
```

Comment:
Eh, it's not *unpublish* submission also. The state transition is: graded(not published) => submitted. 

Any other naming suggestions ?